src/pybind/mgr/nfs/export.py

```diff
@@ -803,14 +825,25 @@ def _apply_export(
                 log.debug("export %s pseudo %s -> %s",
                           old_export.export_id, old_export.pseudo, new_export_dict['pseudo'])
 
+        fsal_dict = new_export_dict.get('fsal')
+        if fsal_dict and fsal_dict['name'] == NFS_GANESHA_SUPPORTED_FSALS[0]:
+            # Ensure cmount_path is present in CephFS FSAL block
+            if not fsal_dict.get('cmount_path'):
+                if old_export:
+                    new_export_dict['fsal']['cmount_path'] = old_export.fsal.cmount_path
```

Comment:
is it possible `old_export.fsal.cmount_path` is empty?

Reply:
> is it possible `old_export.fsal.cmount_path` is empty?

No! We ensure on every change it's atleast = `/` as default if not provided any value